docs/setup_vscode.md

```diff
@@ -143,7 +146,7 @@ $ cd p1-stats
 | `Project 1 Stats/` | `p1-stats/` |
 
 </div>
-
+  
```

Comment:
```suggestion

```

Reply:
This is a good reminder to check the diff on your own PRs.  An ideal PR has the changes you need for the new feature, but not any other distracting changes.